vcluster/third-party-integrations/rancher/install-rancher-integration.mdx

```diff
@@ -21,8 +21,8 @@ This guide explains how to install the vCluster Rancher Operator.
 Ensure you have the following prerequisites before installing the vCluster Rancher Operator:
 
 - Admin permissions for the local cluster. You must be able to add Helm repositories and install charts in the local cluster.
-- A running Rancher local cluster selected for installation
-- vCluster installed and at least one connected downstream cluster where virtual cluster are to be deployed.
+- A running Rancher local cluster selected for installation. Please note that the docker install is not support for production use cases.
+- At least one connected downstream cluster where virtual cluster are to be deployed.
```

Comment:
Connected to what? Platform or Rancher? I think a user would think platform.
```suggestion
- At least one connected downstream cluster where virtual clusters are to be deployed.
```

Reply:
I didn't actually write that part, but we could remove connected, it just refers to downstream cluster.